starship/src/modules/time.rs

```diff
@@ -30,8 +35,33 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         time_format
     );
 
-    let local: DateTime<Local> = Local::now();
-    let formatted_time_string = format_time(&time_format, local);
+    let local_time: DateTime<Local> = Local::now();
+    log::trace!("Local time now is {}", local_time);
+
+    let formatted_time_string: String;
```

Comment:
you don't need to end the assignment here. It's a simple expression, so you can do
```rust
let formatted_time_string: String = if utc_time_offset_str == "local" {
  format_time(&time_format, local_time)
} else {
...
};
```

Reply:
Oh cool. Sorry, I'm pretty new to rust so I'm not aware of things like this. I'll update it. :)

Reply:
No worries. I'm new myself. I just have a head start from Scala 😉

Reply:
I've implemented this pattern. Thanks for pointing it out! Really cool. :smile: